terraform/aurora-mysql/bind/versions.tf

```diff
@@ -4,6 +4,10 @@ terraform {
       source  = "hashicorp/mysql"
```

Comment:
Is this still required?

Reply:
Good question @FelisiaM, I initially thought it wasn’t necessary, but then I changed opinion understanding that they can have applications with the service in beta. Do you think we should eliminate and simplify it? After all, it’s a service that’s supposed to be in beta. I’m waiting for your answer to eliminate it.

Reply:
I worry about what the plan will be in the future to remove it and will we get stuck with it for a long time.
Our options are
1. Remove it now and see if anyone actually was using it. If they complain we put it back as a result.
2. Leave it in indefinitely (ideally until we know everyone has upgraded)

Given our current knowledge about the usage of this service, I would go with the riskier option 1.